test/demo/contextListener.test.ts

```diff
@@ -54,7 +54,13 @@ describe('Context listeners and broadcasting', () => {
                 expect(receivedContexts[0]).toEqual(validContext);
             });
 
-            test.todo('When broadcast is called from the same app [behavior TBD - do apps receive their own broadcasts?]');
+            test('When broadcast is called from the same app', async () => {
```

Comment:
Just a quick note on the test titles. It looks like you just removed the placeholder part of the name but didn't replace it with anything. If you look at the others we have something along the lines of `When doing this thing, that stuff will happen`, so for this case I would prefer something like `When broadcast is called from the same app the listener is not triggered`